custom/auth/ante/ante.go

```diff
@@ -64,7 +70,8 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 		cosmosante.NewConsumeGasForTxSizeDecorator(options.AccountKeeper),
 		cosmosante.NewDeductFeeDecorator(options.AccountKeeper, options.BankKeeper, options.FeegrantKeeper),
 		NewBurnTaxFeeDecorator(options.AccountKeeper, options.TreasuryKeeper, options.BankKeeper, options.DistributionKeeper), // burn tax proceeds
-		cosmosante.NewSetPubKeyDecorator(options.AccountKeeper),                                                               // SetPubKeyDecorator must be called before all signature verification decorators
+		feehshareante.NewFeeSharePayoutDecorator(options.BankKeeper, options.FeeShareKeeper),
```

Comment:
Your providing the wrong BankKeeper type to your Decorator: "cannot use options.BankKeeper (variable of type BankKeeper) as "github.com/classic-terra/core/x/feeshare/ante".BankKeeper value in argument to feehshareante.NewFeeSharePayoutDecorator: BankKeeper does not implement "github.com/classic-terra/core/x/feeshare/ante".BankKeeper (missing method SendCoinsFromModuleToAccount)"